clients/js/src/index.ts

```diff
@@ -77,13 +78,15 @@ export class CohereEmbeddingFunction {
       );
     }
     this.api_key = cohere_api_key;
+    this.model = cohere_model || "large";
```

Comment:
I couldn't find it - but is this the current default embedding model? I just want to make sure we dont introduce a breaking change

Reply:
Double checked✅ "large" is the default model parameter.

Source: [[https://docs.cohere.com/reference/embed]]()